src/configs/tiles.json

```diff
@@ -1,38 +1,83 @@
 [
-  {
-    "type": 0,
-    "amount": 7
-  },
-  {
-    "type": 1,
-    "amount": 10
-  },
-  {
-    "type": 2,
-    "amount": 6
-  },
-  {
-    "type": 3,
-    "amount": 9
-  },
-  {
-    "type": 4,
-    "amount": 2
-  },
-  {
-    "type": 5,
-    "amount": 2
-  },
-  {
-    "type": 6,
-    "amount": 2
-  },
-  {
-    "type": 7,
-    "amount": 1
-  },
-  {
-    "type": 8,
-    "amount": 2
-  }
+    {
+        "type": 0,
+        "amount": 7,
+        "connections": [
+            [0, 1, 0],
+            [0, 1, 0],
+            [0, 1, 0]
+        ]
```

Comment:
I assume this matrix represents a path going from top to bottom
To check the top top middle grid (1) i will have to call something like connections[0][1]?

Reply:
yes, if i understand your question correctly.

Reply:
good just want to make sure the path is represented as we see it